modules/xcode/xcode_common.lua

```diff
@@ -981,17 +981,53 @@
 
 
 	function xcode.XCBuildConfiguration_CLanguageStandard(settings, cfg)
-		if cfg.cdialect and cfg.cdialect ~= "Default" then
-			settings['GCC_C_LANGUAGE_STANDARD'] = cfg.cdialect
-		else
-			settings['GCC_C_LANGUAGE_STANDARD'] = 'gnu99'
+		-- if no cppdialect is provided, don't set C dialect
+		-- projects without C dialect set will use compiler default
+		if not cfg.cdialect then
+			return
+		end
+
+		local cLanguageStandards = {
```

Comment:
It would be kind of nice to have these exist outside of the function. It makes extending or modifying them much easier instead of overriding an entire function, you just `xcode.cLanguageStandards["C20"] = "c20"`.